opnode/rollup/derive/invert.go

```diff
@@ -1,26 +1,58 @@
 package derive
 
 import (
-	"encoding/binary"
+	"errors"
 	"fmt"
 	"math/big"
 
 	"github.com/ethereum/go-ethereum/common"
 )
 
+// setL1BlockValues(uint256 _number, uint256 _timestamp, uint256 _basefee, bytes32 _hash)
```

Comment:
Will we want to add `Random` here as well? I know that is out of scope for this PR, but it seems like doing so eventually will be painful as in a lot of code to change to handle it

Reply:
It's already part of the L2 payload-attributes, and exposed via the `RANDOM` opcode (since l2 doesn't have its own randomness beacon like L1 does post-merge)